desktop/extensions-sdk/dev-tools/api/reference/interfaces/Dialog.md

```diff
@@ -1,19 +1,11 @@
 ---
-title: Docker extension API reference
+title: Dialog
```

Comment:
Wondering; were these files re-generated or were these manual edits?

I also notice that there's no redirects for the old locations (slightly wondering if it's worth doing the move or if we should keep them at the old location)

Reply:
They were manual edits. I'd prefer to do the move and then create a separate PR to add the redirects of the old locations.

Reply:
ISTR these files were generated from source; if that's indeed the case, we should look how we can update the generate process to produce the files with the same changes, otherwise those changes will be reverted the next time we update with the generated docs; @gtardif could you confirm / have a look?

Reply:
Is this a blocker for merging this PR?

Reply:
I think we should at least know what will happen if those docs get updated yes, otherwise the changes will be reverted again (not sure exactly what options we have to make the generated docs match what we want it to look like here 😅)

Reply:
Yes indeed, these are generated pages (from the pinata repo). We can look at improving the generated docs form the SDK APIs, in the pinata repo, but I suggest we do this separately from this PR, so we can move forward with this one (so maybe just revert the changes from files in /api/reference in this PR, and do changes in API reference files separately)